src/lib/viewers/BaseViewer.js

```diff
@@ -560,12 +568,28 @@ class BaseViewer extends EventEmitter {
      * @return {void}
      */
     resize() {
+        this.isSmall = this.isSmallContainer();
```

Comment:
It would be nice to have this be a bit more extensible, where it could support a map of breakpoints to css class names.